src/invidious/comments.cr

```diff
@@ -554,6 +554,32 @@ def fill_links(html, scheme, host)
   return html.to_xml(options: XML::SaveOptions::NO_DECL)
 end
 
+def text_to_parsed_content(text : String) : JSON::Any
```

Comment:
Can you please lighten the code a bit (space out the different chunks) and add comments, please?